### PR TITLE
Edited avalanche-cli import

### PR DESCRIPTION
**Pull request by holisticode**
The import command has been extended to be a command with two subcommands, one pre-existing and a new one.

### CODE REVIEW

File: docs/subnets/reference-cli-commands.md
Comment (yulin-dong) [inline docs/subnets/reference-cli-commands.md:245]: Title cases on heading

File: docs/subnets/reference-cli-commands.md
Comment (yulin-dong) [inline docs/subnets/reference-cli-commands.md:247]:
```suggestion
The `subnet import public` command will import a Subnet configuration from a running network.
```

File: docs/subnets/reference-cli-commands.md
Comment (yulin-dong) [inline docs/subnets/reference-cli-commands.md:250]:
```suggestion
By default, an imported Subnet will not overwrite an existing Subnet with the same name. 
```
All words Subnet should have first letter upper-cased, except in the context of a command, path or logs. See https://github.com/ava-labs/avalanche-docs/blob/master/style-guide.md#style-guide

**Review by yulin-dong — CHANGES_REQUESTED** [review]

File: docs/subnets/reference-cli-commands.md
Comment (yulin-dong) [inline docs/subnets/reference-cli-commands.md:217]:
```suggestion
#### Import From a File
```

File: docs/subnets/reference-cli-commands.md
Comment (holisticode) [reply docs/subnets/reference-cli-commands.md:245]: I was surprised to see that titlecase.com suggests 
"Import from a Public Network" instead of what I would have instinctively applied: "Import From A Public Network". I went for the former.

Comment (yulin-dong) [reply docs/subnets/reference-cli-commands.md:245]: There are two outputs:
Screen Shot 2022-12-05 at 9 42 47 AM

File: docs/subnets/reference-cli-commands.md
Comment (holisticode) [reply docs/subnets/reference-cli-commands.md:217]: Actually titlecase.com suggests "Import from a File", which is what I have chosen.

File: docs/subnets/reference-cli-commands.md
Comment (yulin-dong) [inline docs/subnets/reference-cli-commands.md:211]:
```suggestion
The `subnet import` command imports configurations into Avalanche-CLI.
```

File: docs/subnets/reference-cli-commands.md
Comment (yulin-dong) [inline docs/subnets/reference-cli-commands.md:215]:
```suggestion
(e.g. created manually or with the deprecated Subnet-CLI)
```

**Review by yulin-dong — CHANGES_REQUESTED** [review]

**Comment by dasconnor** [comment]
Let's wait to merge this till after my style check PR is merged

**Review by yulin-dong — APPROVED** [review]

**Pull request merged**
